_data/alumnis.yaml

```diff
@@ -0,0 +1,245 @@
+- name: Kumar Ranjan
+  title: Co-Founder
+  designation: team leader (March, 2014 - July, 2015)
+  image: kumar-ranjan.jpg
+  facebook: https://www.facebook.com/thelostmechanic
+  mail: 
+  linkedin: 
+  github: 
+
+- name: Roshan Prakash
+  title: Co-Founder
+  designation: Team leader (July, 2015 - April, 2016)
+  image: roshan-prakash.jpg
+  facebook: 
+  mail: 
+  linkedin: https://in.linkedin.com/in/roshan-prakash-042ab9b1
+  github: 
+
+#title=position they held in the team
+#designation=place/designation of their work

```

Comment:
Also, move this comment to the top of the file.

Reply:
When someone hovers over an alumnus' image, they get to see the current designation in place of the team position, the way it currently happens for Mr Shreyansh Daftry.
For members whose position we don't know, instead of keeping designation blank, we kept it the same as the team title. This way it doesn't change when someone hovers over their name. Should I change it to blank only?